packages/fast-tooling-react/src/form/controls/control.section.spec.tsx

```diff
@@ -1,15 +1,17 @@
 import React from "react";
 import Adapter from "enzyme-adapter-react-16";
 import { configure, mount } from "enzyme";
-import StyledFormSection, { FormSection } from "./form-section";
-import { FormSectionClassNameContract, FormSectionProps } from "./form-section.props";
+import StyledSectionControl, { SectionControl } from "./control.section";
+import {
+    SectionControlClassNameContract,
+    SectionControlProps,
+} from "./control.section.props";
 import { ContextComponent, DragDropContext } from "react-dnd";
 import HTML5Backend from "react-dnd-html5-backend";
 
-const TestFormSection: typeof StyledFormSection & ContextComponent<any> = DragDropContext(
-    HTML5Backend
-)(StyledFormSection);
-import { controls } from "./form-control-switch.spec";
+const TestSectionControl: typeof StyledSectionControl &
```

Comment:
This is totally just preference - having this mixed between imports was a bit confusing from a readability standpoint. If there's a good reason, feel free to ignore this comment.

Reply:
That's valid, I'm not sure how it happened actually.